udata/templates/dataset/display.html

```diff
@@ -148,13 +148,13 @@ <h4 class="list-group-item-heading">
                             {{ share_btn(dataset.title) }}
                         </div>
                         <div class="btn-group btn-group-sm">
-                            {{ integrate_btn(dataset.id) }}
+                            {{ integrate_btn(dataset) }}
                         </div>
                         <div class="btn-group btn-group-sm">
                             {{ issues_btn(dataset) }}
                         </div>
                     </div>
-                </div>
+                </div></div>
```

Comment:
Is that div opened within the button bar? Otherwise put it after the comment below.